znnl/models/jax_model.py

```diff
@@ -123,7 +122,7 @@ def __init__(
 
         # Prepare NTK calculation
         self.empirical_ntk = nt.batch(
-            nt.empirical_ntk_fn(f=self._ntk_apply_fn, trace_axes=trace_axes),
+            nt.empirical_ntk_fn(f=jax.jit(self._ntk_apply_fn), trace_axes=trace_axes),
```

Comment:
I think this is from an older version of main. 
Can you remove it?

Reply:
Done in [c5e1170](https://github.com/zincware/ZnNL/pull/117/commits/c5e1170613ec80b29d8b1b0acd2b5801520a8104)